### PR TITLE
test: add OAuth integration test

### LINKED ISSUE #34: Write integration test for OAuth redirect
The integration test needs to invoke `terraform init`, `terraform apply`, and `terraform destroy`.

### PR DESCRIPTION
**Pull request by jasonwalsh**
Resolves #34

### CODE REVIEW

**Comment by brh55** [comment]
Pretty neat stuff J. +1

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import pytest
+import os
+
+from glob import glob
+from zipfile import ZipFile
+from python_terraform import Terraform
+
+
+def make_zip_file():
+    os.chdir('functions')
```
Comment (jasonwalsh) [inline tests/integration/conftest.py]: I think the `make_zip_file` function should change directory back o the original directory once the archive is created.

For example:

```python
def make_zip_file():
    oldpwd = os.getcwd()
    os.chdir('functions')
    ...
    zip_file.close()
    os.chdir(oldpwd)
```

Comment (brh55) [reply tests/integration/conftest.py]: Yes please do that ⭕️ Scootz

**Pull request merged**